src/PageSettings.cpp

```
@@ -4,13 +4,78 @@
+
+  case PageLayout::Overlay::RASP:
+    builder.Append(", RASP");
+    if (rasp != nullptr &&
+        layout.rasp_field >= 0 &&
+        unsigned(layout.rasp_field) < rasp->GetItemCount()) {
+      const auto &item = rasp->GetItemInfo(layout.rasp_field);
+      const char *label = item.label != nullptr
+        ? gettext(item.label)
+        : item.name;
+      if (label != nullptr && *label != '\0') {
+        builder.Append(' ');
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)